Makefile

```diff
@@ -130,6 +131,30 @@ install-proto-bin: install-glide
 		go install $(m3db_package)/$(vendor_prefix)/$(protoc_go_package)    \
 	)
 
+install-stringer:
+		@which stringer > /dev/null || go get golang.org/x/tools/cmd/stringer
+		@which stringer > /dev/null || (echo "stringer install failed" && exit 1)
+
+install-goreleaser: install-stringer
+		@which goreleaser > /dev/null || (go get -d github.com/goreleaser/goreleaser && \
+		  cd $(GOPATH)/src/github.com/goreleaser/goreleaser && \
+			git checkout v0.76.1 && \
```

Comment:
nit: mind using a var for this `GORELEASER_VERSION := ...`

Reply:
sure